packages/teams-js/src/public/search.ts

```diff
@@ -127,4 +145,74 @@ export namespace search {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.search ? true : false;
   }
+
+  /**
+   * Clear the host M365 application's search box
+   *
+   * @beta
+   */
+  export function closeSearch(): Promise<void> {
+    return new Promise<void>((resolve) => {
+      ensureInitialized(runtime, FrameContexts.content);
+      if (!isSupported()) {
+        throw new Error('Not supported');
+      }
+
+      resolve(sendAndHandleError('search.closeSearch', version));
+    });
+  }
+
+  /**
+   * @hidden
+   * Undocumented function used to set a mock window for unit tests
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  export function _initialize(hostWindow: any): void {
```

Comment:
This doesn't seem like a function that should live in the search capability. I assume you're only calling it from your unit tests, in which case you should probably move it there. There's nothing search specific about this and putting it here will add to the package size for external developers.

Reply:
Yeah it was also for an old version of tests will delete it